config-model/src/main/java/com/yahoo/config/model/deploy/DeployProperties.java

```diff
@@ -18,14 +21,16 @@
     private final boolean multitenant;
     private final ApplicationId applicationId;
     private final List<ConfigServerSpec> serverSpecs = new ArrayList<>();
+    private final URI loadBalancerAddress;
     private final boolean hostedVespa;
     private final Version vespaVersion;
     private final Zone zone;
 
     private DeployProperties(boolean multitenant,
                              ApplicationId applicationId,
                              List<ConfigServerSpec> configServerSpecs,
-                             boolean hostedVespa, Version vespaVersion, Zone zone) {
+                             URI loadBalancerAddress, boolean hostedVespa, Version vespaVersion, Zone zone) {
```

Comment:
It looks strange to have a list of params on one line compared to the above.

Reply:
fixed